### DISCUSSION TITLE
Can't install older version of jaxlib

### DISCUSSION THREAD

**Discussion by Kieran-Loehr** [jax-ml/jax#19341] [Q&A] [answered]
I am trying to get an older version of jaxlib that would be compatible cuDNN 8.2. I tried following the instructions here https://jax.readthedocs.io/en/latest/installation.html#installing-older-jaxlib-wheels but as shown below I can still only see versions of 0.4.17 and later. Is there a different way to get older versions of jaxlib? 
![image](https://github.com/google/jax/assets/106624136/8f858a86-c558-4c52-9cd5-b20325907131)

**Comment by jakevdp** [comment, marked as answer]
One issue is that the URL you are using is the one for CPU wheels; if you want GPU wheels you should use the URL in the GPU installation instructions further up on that page: https://storage.googleapis.com/jax-releases/jax_cuda_releases.html

Since you're only seeing version 0.4.17 or later, I suspect you are probably using Python 3.12. v0.4.17 was the first wheel release built for Python 3.12; if you want to use an older release, you'll have to downgrade to an older Python version. JAX v0.3.25 supports Python 3.7, 3.8, 3.9, 3.10, or 3.11.

**Reply by Kieran-Loehr** [reply]
Thanks so much, downgrading to an older Python version worked.
